apps/api/src/functions/resources/app.js

```diff
@@ -0,0 +1,17 @@
+const bodyParser = require("body-parser");
+const cors = require("cors");
+const express = require("express");
+const router = require("./router.js");
+
+const app = express();
+
+app.use(cors());
+app.use(bodyParser.json());
```

Comment:
```suggestion
app.use(express.json());
```
All the middleware did get stripped out of express but `bodyParser` got [added back in](https://github.com/expressjs/express/pull/3423) and we can use it like this - we could `npm rm body-parser` (and delete line 1)